FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -52,11 +52,17 @@ public void ExecuteSelectTest(string commandText)
             executeTest(SYSTEM_CONNECTION_STRING, commandText, null, true);
         }
 
-        [Ignore("sleepEachRow function is currently not supported on staging")]
-        [TestCase("select sleepEachRow(1) from numbers(5)")]
-        public void ExecuteSetTest(string commandText)
+        [Test]
+        [Category("slow")]
+        public void ExecuteLongTest()
         {
-            executeTest(SYSTEM_CONNECTION_STRING, commandText, "SET use_standard_sql=0", true);
+            using var conn = new FireboltConnection(SYSTEM_CONNECTION_STRING);
+            conn.Open();
+            DbCommand command = conn.CreateCommand();
+            command.CommandTimeout = 0; // make command timeout unlimited
```

Comment:
I'd still add a large timeout here, just to be safe. This way if the query misbehaves in any way we won't have tests running forever.

Reply:
Good point. I have added timeout.